entity-service/src/main/kotlin/com/egm/stellio/entity/web/EntityOperationHandler.kt

```diff
@@ -163,6 +167,67 @@ class EntityOperationHandler(
             ResponseEntity.status(HttpStatus.MULTI_STATUS).body(batchOperationResult)
     }
 
+    /**
+     * Implements 6.16.3.1 - Update Batch of Entities
+     */
+    @PostMapping("/update", consumes = [MediaType.APPLICATION_JSON_VALUE, JSON_LD_CONTENT_TYPE])
+    suspend fun update(
+        @RequestHeader httpHeaders: HttpHeaders,
+        @RequestBody requestBody: Mono<String>,
+        @RequestParam options: Optional<String>
+    ): ResponseEntity<*> {
+        val userId = extractSubjectOrEmpty().awaitFirst()
+        val body = requestBody.awaitFirst()
+        checkContext(httpHeaders, body)
+        val context = getContextFromLinkHeader(httpHeaders.getOrEmpty(HttpHeaders.LINK))
+        val disallowOverwrite = options.map { it == "noOverwrite" }.orElse(false)
+
+        val (extractedEntities, jsonLdEntities, ngsiLdEntities) =
+            extractAndParseBatchOfEntities(body, context, httpHeaders.contentType)
+        val (existingEntities, newEntities) = entityOperationService.splitEntitiesByExistence(ngsiLdEntities)
```

Comment:
if an entity does not exist, it should be added as an error element in the response (as described in 5.6.3, referenced in 5.6.9.4)